src/plugins/intel_cpu/tests/functional/subgraph_tests/src/gather_weights_decompression.cpp

```diff
@@ -0,0 +1,193 @@
+ *                 \      /
+ *                  MatMul(OP)  (Add MatMul in order to test OP==bf16 in SPR)
+ */
+
+struct InputAndWeigthsShapeParams {
+    InputAndWeigthsShapeParams() = default;
+    InputAndWeigthsShapeParams(InputShape _data_shape, ov::Shape _weights_shape)
+        : data_shape(std::move(_data_shape)),
+          weights_shape(std::move(_weights_shape)) {}
+
+    InputShape data_shape;
+    ov::Shape weights_shape;
+};
+
+using GatherWeightsDecompressParams = std::tuple<InputAndWeigthsShapeParams,
+                                                 ov::AnyMap,  // additional config
+                                                 fusingSpecificParams,
+                                                 bool>;  // should use decompression implementation
+
+class GatherWeightsDecompression : public testing::WithParamInterface<GatherWeightsDecompressParams>,
+                                virtual public SubgraphBaseTest,
+                                public CpuTestWithFusing {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<GatherWeightsDecompressParams> obj) {
+        InputAndWeigthsShapeParams shape_params;
+        ov::AnyMap additional_config;
+        fusingSpecificParams fusing_params;
+        bool should_fuse;
+
+        std::tie(shape_params, additional_config, fusing_params, should_fuse) = obj.param;
+
+        std::ostringstream result;
+        result << "data_shape=" << shape_params.data_shape << "_";
+        result << "weights_shape=" << shape_params.weights_shape << "_";
+
+        result << "config=(";
+        for (const auto& configEntry : additional_config) {
+            result << configEntry.first << ", " << configEntry.second.as<std::string>() << ":";
+        }
+        result << ")";
+        result << CpuTestWithFusing::getTestCaseName(fusing_params);
+
+        return result.str();
+    }
+
+protected:
+    std::shared_ptr<ov::Node> initDecompressionWeights(const ov::Shape& weights_shape,
+                                                       const ov::element::Type weights_precision) {
+        auto weights =
+            ov::test::utils::deprecated::make_constant<int8_t>(weights_precision, weights_shape, {}, true, 255);
+        weights->set_friendly_name("Compressed_weights");
+        auto weights_convert = std::make_shared<ov::op::v0::Convert>(weights, ov::element::f16);
+
+        std::shared_ptr<ov::Node> zp_const =
+            ov::test::utils::deprecated::make_constant<float>(ov::element::u8,
+                                                              ov::Shape{weights_shape[0], 1},
+                                                              {},
+                                                              true);
+        auto zp_convert = std::make_shared<ov::op::v0::Convert>(zp_const, ov::element::f16);
+
+        std::shared_ptr<ov::Node> scale_const =
+            ov::test::utils::deprecated::make_constant<float>(ov::element::f16,
```

Comment:
please replace all calls of  `ov::test::utils::deprecated::make_constant`  with `ov::test::utils::make_constant` instead (

Reply:
Done